### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**
There might be some issues with auto-replacement of "імʼя" to "назва"

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -5439,7 +5438,7 @@ msgstr ""
 #: builtin/clone.c:1162 builtin/remote.c:204 builtin/remote.c:731
 #, c-format
 msgid "'%s' is not a valid remote name"
-msgstr "'%s' не є дійсним імʼям віддаленого сховища"
+msgstr "'%s' не є дійсним назвам віддаленого сховища"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "\"%s\" не є дійсною назвою віддаленого сховища"
```

File: po/uk.po [hunk 2]
```diff
@@ -12154,7 +12152,7 @@ msgstr ""
 "\n"
 "    git remote add <ім'я> <адреса>\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"    git remote add <назва> <адреса>\n"
```

File: po/uk.po [hunk 3]
```diff
@@ -12154,7 +12152,7 @@ msgstr ""
...
 "\n"
-"а потім виконайте надсилання, використовуючи імʼя віддаленого сховища\n"
+"а потім виконайте надсилання, використовуючи назва віддаленого сховища\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"а потім виконайте надсилання, використовуючи назву віддаленого сховища\n"
```

File: po/uk.po [hunk 4]
```diff
@@ -13647,7 +13645,7 @@ msgstr ""
 #: builtin/replace.c:163
 #, c-format
 msgid "'%s' is not a valid ref name"
-msgstr "\"%s\" не є припустимим імʼям посилання"
+msgstr "\"%s\" не є припустимим назвам посилання"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "\"%s\" не є припустимою назвою посилання"
```

File: po/uk.po [hunk 5]
```diff
@@ -13690,7 +13688,7 @@ msgstr ""
 
 #: builtin/replace.c:298
 msgid "mktree did not return an object name"
-msgstr "mktree не повернув імʼя обʼєкта"
+msgstr "mktree не повернув назва обʼєкта"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "mktree не повернув назву обʼєкта"
```

File: po/uk.po [hunk 6]
```diff
@@ -14428,7 +14426,7 @@ msgstr ""
 
 #: builtin/show-branch.c:661
 msgid "suppress naming strings"
-msgstr "не показувати імʼя"
+msgstr "не показувати назва"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не показувати назву"
```

File: po/uk.po [hunk 7]
```diff
@@ -18254,7 +18252,7 @@ msgstr ""
 #: config.c:3076
 #, c-format
 msgid "invalid section name '%s'"
-msgstr "неприпустиме імʼя секції \"%s\""
+msgstr "неприпустиме назва секції \"%s\""
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "неприпустима назва секції \"%s\""
```

File: po/uk.po [hunk 8]
```diff
@@ -18309,7 +18307,7 @@ msgstr "не вдалося встановити \"%s\" в \"%s\""
 #: config.c:3739
 #, c-format
 msgid "invalid section name: %s"
-msgstr "неприпустиме імʼя секції: %s"
+msgstr "неприпустиме назва секції: %s"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "неприпустима назва секції: %s"
```

File: po/uk.po [hunk 9]
```diff
@@ -20292,7 +20290,7 @@ msgstr ""
 #: ident.c:503
 #, c-format
 msgid "unable to auto-detect name (got '%s')"
-msgstr "не вдалося автоматично визначити імʼя (отримано \"%s\")"
+msgstr "не вдалося автоматично визначити назва (отримано \"%s\")"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не вдалося автоматично визначити назву (отримано \"%s\")"
```

File: po/uk.po [hunk 10]
```diff
@@ -21564,7 +21562,7 @@ msgstr ""
 #: object-name.c:2049
 #, c-format
 msgid "invalid object name '%.*s'."
-msgstr "неприпустиме імʼя обʼєкта \"%.*s\"."
+msgstr "неприпустиме назва обʼєкта \"%.*s\"."
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "неприпустима назва обʼєкта \"%.*s\"."
```

File: po/uk.po [hunk 11]
```diff
@@ -22175,7 +22173,7 @@ msgstr "promisor-remote: не вдалося закрити stdin для під
 #: promisor-remote.c:58
 #, c-format
 msgid "promisor remote name cannot begin with '/': %s"
-msgstr "імʼя віддаленого promisor не може починатися з '/': %s"
+msgstr "назва віддаленого promisor не може починатися з '/': %s"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "назва віддаленого promisor не може починатися з \"/\": %s"
```

File: po/uk.po [hunk 12]
```diff
@@ -22789,7 +22787,7 @@ msgstr "формат: %%(end) частка відсутня"
 #: ref-filter.c:2855
 #, c-format
 msgid "malformed object name %s"
-msgstr "неправильно сформованe імʼя обʼєкта %s"
+msgstr "неправильно сформованe назва обʼєкта %s"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "неправильно сформована назва обʼєкта %s"
```

File: po/uk.po [hunk 13]
```diff
@@ -23368,7 +23366,7 @@ msgstr "  (використовуйте \"git pull\", щоб злити відд
 #: remote.c:2518
 #, c-format
 msgid "cannot parse expected object name '%s'"
-msgstr "неможливо розібрати очікуване імʼя обʼєкту '%s'"
+msgstr "неможливо розібрати очікувану назву обʼєкту '%s'"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "неможливо розібрати очікувану назву обʼєкту \"%s\""
```

**Review by kasya — COMMENTED** [review]
👍🏼

File: po/uk.po (hunk 4 above)
Comment (arkid15r) [reply po/uk.po]: 😵‍💫

**Review by arkid15r — COMMENTED** [review]
Thanks for the review!

**Pull request merged**